test_regress/t/t_debug_emitv.out

```diff
@@ -7,79 +7,79 @@ module Vt_debug_emitv_t;
     ???? // ENUMITEM 'ZERO'
     32'h0
     ???? // ENUMITEM 'ONE'
-    32'h1int signed [31:0] struct packed 
-    {int signed [31:0]  a;}logic signed [2:0] struct 
+    32'h1int signedstruct packed 
+    {int signed a;}logic signed [2:0] struct 
     {logic signed [2:0]  a;}logicunion 
     {logic a;}struct packed 
-    {int signed [31:0]  a;}bit [31:0] const struct packed 
-    {int signed [31:0]  a;}const struct packed 
-    {int signed [31:0]  a;}[0:2]struct 
+    {int signed a;}bit [31:0] const struct packed 
+    {int signed a;}const struct packed 
+    {int signed a;}[0:2]struct 
     {logic signed [2:0]  a;}union 
-    {logic a;}int signed [31:0] int signed [31:0] [0:2]logic [15:0] logic [15:0] logic [15:0] int signed [31:0] int signed [31:0] int signed [31:0] 
+    {logic a;}int signedint signed[0:2]logic [15:0] logic [15:0] logic [15:0] int signedint signedint signed
```

Comment:
`signedint signed`? ;)

Reply:
Ok,, this is unrelated to your change, we'll defer.

Reply:
Thanks. I think this is a problem with missing support for more complex types in V3EmitV, our target would be to emit all verilog nodes back to verilog code, but it suffices for simple hierarchical verilation use-cases for now.